cli/run_util.go

```diff
@@ -278,9 +279,14 @@ func runStep(step stepmanModels.StepModel, stepIDData models.StepIDData, stepDir
 		}
 
 		if options.IsTemplate != nil && *options.IsTemplate {
-			envList, err := bitrise.EnvmanJSONPrint(bitrise.InputEnvstorePath)
+			outStr, err := bitrise.EnvmanJSONPrint(bitrise.InputEnvstorePath)
 			if err != nil {
-				return 1, []envmanModels.EnvironmentItemModel{}, err
+				return 1, []envmanModels.EnvironmentItemModel{}, fmt.Errorf("EnvmanJSONPrint failed, err: %s", err)
+			}
+
+			envList := envmanModels.EnvsJSONListModel{}
```

Comment:
move this into a constructor method, which accepts a JSON `string` and return the parsed `envmanModels.EnvsJSONListModel` ?